features/providers/civil_application_journey.feature

```diff
@@ -296,6 +296,9 @@ Feature: Civil application journeys
     Then I click 'Save and continue'
     Then I should be on a page showing "Covered under an emergency certificate"
     Then I should be on a page showing "Covered under a substantive certificate"
+    When I choose 'Yes'
+    And I enter a emergency cost requested '5000'
+    And I enter legal aid application emergency cost reasons field 'This is why I require extra funding'
```

Comment:
lgtm. Could you have an integration test for the updated merits section?

Reply:
The merits section stuff is hidden behind an `<if read_only>` check. This means it is only visible on the caseworker merits report and not visible to the user on the `check_merits_answers` page